src/Connection.js

```diff
@@ -27,7 +28,21 @@ export class Connection {
     this.wsInBuffer = new Buffer(0);
     this.handshakeComplete = false;
     this.isClosed = false;
-    this.dbSocket = net.createConnection(dbPort, dbHost);
+    let options = {
+      host: dbHost,
+      port: dbPort
+    };
+    if (dbSsl === null) {
+      this.dbSocket = net.connect(options);
+    } else if (typeof dbSsl === 'boolean' && dbSsl) {
+      this.dbSocket = tls.connect(options);
+    } else if (typeof dbSsl === 'object') {
+      if (typeof dbSsl.ca !== 'undefined') {
+        options.ca = dbSsl.ca;
+      }
```

Comment:
Instead of explicitly just copying over the `ca` field, let's copy over anything they specify, since `tls.connect` theoretically supports a lot more customizable options. If you do `options = {...dbSsl, ...options}` (ES6 spread syntax), it should combine them into one object.